components/clarinet-cli/src/generate/project.rs

```diff
@@ -341,14 +341,14 @@ disable_stacks_api = false
 # subnet_api_image_url = "{default_subnet_api_image}"
 # subnet_api_postgres_database = "subnet_api"
 
-# For testing in epoch 2.1 / using Clarity2
 # epoch_2_0 = {DEFAULT_EPOCH_2_0}
 # epoch_2_05 = {DEFAULT_EPOCH_2_05}
 # epoch_2_1 = {DEFAULT_EPOCH_2_1}
 # epoch_2_2 = {DEFAULT_EPOCH_2_2}
 # epoch_2_3 = {DEFAULT_EPOCH_2_3}
 # epoch_2_4 = {DEFAULT_EPOCH_2_4}
 # epoch_2_5 = {DEFAULT_EPOCH_2_5}
+# epoch_3_0 = 144
```

Comment:
We want to add this commented line for visibility/discoverability.
But keep it hard coded for now because the DEFAULT_EPOCH_3_0 will remain very high until epoch 3.0 lands on mainnet